prow/test-infra-cleanup-cluster.sh

```diff
@@ -27,4 +27,4 @@ set -u
 set -x
 
 echo "=== Cleaning up cluster ==="
-./scripts/cleanup-cluster -h 2
+./scripts/cleanup-cluster -c '' -h 2
```

Comment:
CLUSTER_NAME is set to e2e-testing by default. Because we are mounting kubeconfig we don't need to call get-credentials. I was also getting an error before changing this
ERROR: (gcloud.container.clusters.get-credentials) ResponseError: code=403, message=Required "container.clusters.get" permission for "projects/istio-testing/zones/us-west1-a/clusters/e2e-testing".